controllers/toolchaincluster/toolchaincluster_controller_test.go

```diff
@@ -160,6 +207,50 @@ func TestClusterControllerChecks(t *testing.T) {
 	})
 }
 
+func TestGetClusterHealth(t *testing.T) {
```

Comment:
aren't these test cases already covered in the previous tests?

Reply:
These check the checkHealth param usage..(if someday we set it through main(while calling the controller) so  i added them to test that scenario and also they do the integration testing of checking the health check function and toolchain controller  together.